src/backend/launcher.ts

```diff
@@ -964,6 +986,77 @@ async function callRunner(
   return promise
 }
 
+async function stopChildProcesses(childPid: number) {
+  if (isWindows) {
+    logWarning(
+      `Killing all processes spawned by PID ${childPid}`,
+      LogPrefix.Backend
+    )
+
+    try {
+      // Get the list of child processes
+      const getChildProcesses = async (parentPid: number) => {
+        const result = await spawnAsync('powershell.exe', [
```

Comment:
Will this only work on windows?

Reply:
This part yes, at the end of the method you will find the command to mac and linux.